ember/app/styles/components/_top-nav.scss

```diff
@@ -11,23 +11,33 @@
   padding: 0 30px;
   height: var(--navbar-height);
 
-  a {
+  .uk-logo {
     min-height: var(--navbar-height);
   }
 }
 
 /* button nav */
 .button-nav {
   @extend .uk-button-group;
+  @extend .uk-navbar-nav;
 
-  button {
+  list-style: none;
+  gap: 0;
+
+  .ember-view {
```

Comment:
`.ember-view` can be any actionable dom element with an ember action on it. Assign a class to those items and style that class accordingly

Reply:
Instead of assigning a class I styled them with `li a` is this also okay?
https://github.com/adfinis/Outdated/blob/40dcbadc601bde66738389213280918de7e3ab52/ember/app/styles/components/_top-nav.scss#L27-L42

Reply:
Yes, that works as well.